e2e_tests/tests/per_provider/normal_tests/asym_sign_verify.rs

```diff
@@ -59,11 +59,10 @@ fn asym_sign_and_verify_rsa_pkcs() -> Result<()> {
     client.verify_with_rsa_sha256(key_name, HASH.to_vec(), signature)
 }
 
-#[cfg(not(any(feature = "tpm-provider", feature = "trusted-service-provider")))]
```

Comment:
You removed TS provider as well as TPM one in a few places. Was it intentional? It has nothing to do with the PR description.